src/test/java/org/opentripplanner/routing/algorithm/mapping/SnapshotTestBase.java

```diff
@@ -272,7 +274,17 @@ private String createDebugUrlForRequest(RouteRequest request) {
       .atZone(serverContext().transitService().getTimeZone())
       .toLocalDateTime();
 
-    var transitModes = mapModes(request.journey().transit().modes());
+    // TODO: 2022-12-20 filters: this is for REST so there should not be more than one filter
```

Comment:
Could you explain what this has to do with the REST API?

Reply:
With new API there could be more than one transportModes filter specified per request but since REST api does not support it, `request.journey().transit().filters()` should always contain only 1 element.